addons/mail/static/src/models/activity/activity.js

```diff
@@ -167,7 +167,7 @@ function factory(dependencies) {
                     attachment_ids: attachmentIds,
                     feedback,
                 },
-            }));
+            }, { shadow: true }));
```

Comment:
I would revert this one, this is a direct user action for which we don't handle "loading" ourselves.